### DISCUSSION TITLE
What happens when the resolution of L0 & L2 are the same？

### DISCUSSION THREAD

**Discussion by LuvMangorange** [mhm-ufz/mHM#26] [Show and tell]
Hello, everone

I think this is an interesting or useful thing, and I want to share with people who need it, after I knew why “Number of outlets found at level 0:..    0” occured in my example.
From the title, maybe you already know that, yes, when the resolution of L0 & L2 are the same, the number of outlets will be 0 at level 0. This is the focus of this discussion. If you are intersted, please keep reading.

when I encountered this error for the first time, I repeated the steps 3-5 times,  in case I made some mistakes in the morphological data. In the process, I have doubted whether the precision of dem is not good enough, whether the flow direction does not correspond to the domain edge of other data,  whether the flow accumulation ends at the edge of the watershed, whether the outlet is the max value of flow accumulation, etc. But the results were always the same: “Number of outlets found at level 0:..    0”. 
Therefore, I debuged the program. In the code(Figure 1), the number of outlet at L0 is determined by the variable Noutlet. However, the value of Noutlet does not be changed during the operation of subroutine "L11_flow_direction(iDomain) " at the case where routing and input data scale is similar. 
According to the debugging result, I think this error will not affect the running of my example, and keep the progam running while the error message appeares again. After 4 hours, mhm model was finished!（Figure 2)

Figure 1：
![image](https://user-images.githubusercontent.com/34867664/123530538-1a05fe80-d72e-11eb-9246-694a609ae916.png)
Figure 2：
![image](https://user-images.githubusercontent.com/34867664/123530595-d5c72e00-d72e-11eb-9ee5-37eb71f849e2.png)

**Comment by 9202014710** [comment]
Seems cool. Can I hava a discussion with you when you available?

**Comment by rohinikumar82** [comment]
This is great - happy that you have experienced something new. Could you please close this issue; and no worries your experience will be archived and available to other users as well.
